roles/build-release/README.md

```diff
@@ -4,12 +4,8 @@ An Ansible role that builds an Ansible release package with Antsibull and tests
 
 ### Requirements
 
-This role is expected to run directly from an antsibull git repository checkout.
-
-Otherwise:
-
 - ansible-core to run this role
-- poetry to install and run antsibull
+- antsibull needs build Ansible
```

Comment:
``antsibull needs build Ansible`` seems like it could be rephrased, I can't parse what it is meant to say

Reply:
Fixed in 887e058515a47b8e022ef75d00bc96c858957a18.